spec/fixtures/iso-sample-identifiers-v2.txt

```diff
@@ -1,4 +1,4 @@
-ISO 5843/6
+ISO 5843-6
```

Comment:
We should not change this - this is the old official ISO pattern where part is delimited with `/`. We should still parse this pattern but not generate it.